app/bundles/FormBundle/Model/FormModel.php

```diff
@@ -503,17 +503,23 @@ public function generateHtml(Form $entity, $persist = true)
         $lead        = ($this->requestStack->getCurrentRequest()) ? $this->contactTracker->getContact() : null;
         $style       = '';
 
-        if (!empty($theme)) {
```

Comment:
@escopecz I stopped two parts of the code in this file, because they are embedding templates from the theme, this should not be done this way, either we need to use twig file loader to find if the files are there then embed them

Reply:
@mabumusa1 there are more places where the theme is prefixed. For example in https://github.com/mautic/mautic/pull/12115/files#diff-a72d8252cec5327fca1e2760f3572824afc78159017148e44b5c012272803102R37. I'm not clear on whether this theme prefixing is a systematic issue or local to this file.

Reply:
I think it is local, with this method we are using | to find files, which is not supported by twig, we need to use the loader in a specific path to get the file

Reply:
The same is happening in the line I linked. So it's in more places.

Reply:
Let's wait on this PR a bit, I do not have a solution at the moment. I will create another PR to change the templates I am sure about

Reply:
@escopecz I found a solution, it will work after we merge a couple of PRs mainly to prepare the removal of templating helper

Reply:
In #666e51885a776f8abaf5fb1dacb4947af1cce45b I solved all these issues